LibOS/shim/src/bookkeep/shim_vma.c

```diff
@@ -546,6 +557,17 @@ static inline void __shrink_vma (struct shim_vma * vma, void * start, void * end
     assert(vma->start < vma->end);
 }
 
+/*
+ * Update bookkeeping for munmap(). "prev" must be passed into the function as
+ * this immediate precedent vma of the deallocating address, or NULL if no
```

Comment:
`this`? Didn't you mean `the`? Also, `passed as` is confusing and it has a different meaning than intended. Same for `deallocating`.

Suggested version: `"pprev" must point to the vma immediately precedent to the deallocated address, [...]`

Reply:
Comments rewritten.